components/react/src/Components/Selector/Selector.js

```diff
@@ -1,28 +1,33 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import SelectorModal from '../SelectorModal';
 import Category from '../Category';
 import Tag from '../Tag';
 import _ from 'lodash';
 
-const Selector = ({ categories, name, taxonomyId }) => {
+const Selector = ({ categories, name, taxonomyId, addToMediator }) => {
   const [selected, setSelected] = useState([]);
+  useEffect(() => addToMediator(selected.map(getTagId)));
 
   const selectorCategory = (categoryName, id) =>
-    <Category categoryId={id} name={categoryName} click={() => removeCategoryFromSelected(id)} key={'category' + id} />;
+    <Category categoryId={id} name={categoryName} click={() => removeCategoryFromSelected(id)} key={`category${id}`} />;
```

Comment:
This `categoryId` vs `tagId` thing is lame. How can we improve it? It makes this weird and matchmaking kind of hard.